### DISCUSSION TITLE
View card without access

### DISCUSSION THREAD

**Discussion by lubo313** [NemesisRE/kiosk-mode#103] [Q&A]
The user has the right to display two cards. After launch, these two tabs are visible. However, after a while, a card that he is not entitled to is also displayed. Why? 
(google translate)

Lubo

**Comment by elchininet** [comment]
Hi @lubo313,
It is hard to understand what you are referring to. In general `kiosk-mode` is not related to `cards`. So please, attach some screenshots as that can help to understand what you are referring to.
Regards

**Comment by lubo313** [comment]
Hi,

configuration dashboard for user "chatka":

`kiosk_mode:
  user_settings:
    - users:
        - chatka
      hide_sidebar: true
      hide_menubutton: true
      hide_header: false
      hide_edit_dashboard: true
      hide_search: true
      hide_assistant: true
      hide_overflow: true
      kiosk: false
      ignore_entity_settings: true
`

The user has access only for two cards marked in green:
![all_cards](https://github.com/NemesisRE/kiosk-mode/assets/88633150/e80782bd-225c-4845-b78c-f1d765e61a8e)

Card access:
![card_access](https://github.com/NemesisRE/kiosk-mode/assets/88633150/56c16f41-4957-4c19-81da-0a14479dbc01)

Card NO access:
![card_NO_access](https://github.com/NemesisRE/kiosk-mode/assets/88633150/481cbd58-0e17-4350-9189-9b0cc0d6311f)

I'm using Wall-panel, after starting everything works fine, only access cards are displayed. As a saver there is a clock. After a while, if I wake up the panel, the content of the card with access is not displayed, but the one to which this user does not have access (marked in red in the image above). I don't understand why this particular card, when there are a lot of them, is not even the first in the order...
(google translate)

Thanks, Lubo

**Reply by elchininet** [reply]
Hi @lubo313,

You are showing the `kiosk-mode` configuration but later you complain about something that is not related to `kiosk-mode` in any way, but to Home Assistant itself.

Hiding views from users is not a `kiosk-mode` feature, it is a native feature. I really don't know why this is not working for you, but I recommend you to disable all your custom plugins and start from scratch to see if it is still happening, later if it doesn't occur with all the plugins disabled, start enabling custom plugins one by one to see if you can detect where the issue is.

Regards

**Comment by lubo313** [comment]
I did a factory reset of the tablet, installed the app and everything works OK.
Lubo
